crates/typst-library/src/foundations/array.rs

```diff
@@ -350,14 +341,10 @@ impl Array {
         Ok(None)
     }
 
-    /// Create an array consisting of a sequence of numbers.
-    ///
-    /// If you pass just one positional parameter, it is interpreted as the
-    /// `end` of the range. If you pass two, they describe the `start` and `end`
-    /// of the range.
-    ///
-    /// This function is available both in the array function's scope and
-    /// globally.
+    /// 数列で構成される配列を作成します。
+    /// 位置引数を1つだけ渡した場合、それは範囲の`終了`位置と解釈されます。
+    /// 2つ渡した場合は、範囲の`開始`と`終了`位置を示します。
```

Comment:
`end`と`start`は引数名であるため、英語のままの方が望ましいです。
```suggestion
    /// 位置引数を1つだけ渡した場合、それは範囲の`end`と解釈されます。
    /// 2つ渡した場合は、範囲の`start`と`end`を示します。
```